lib/configuration/client_configuration.js

```diff
@@ -150,6 +173,7 @@ function ConfigurationUtil(fsPromisesModule, processModule) {
         )
       );
     } catch (err) {
+      Logger.getInstance().error('File contains correct JSON. Validating the input.');
```

Comment:
is this correct?

Reply:
You right, I missed that one out. Thanks